pywren/serialize/cloudpickle/cloudpickle.py

```diff
@@ -12,7 +12,7 @@
 <https://web.archive.org/web/20140626004012/http://www.picloud.com/>`_.
 
```

Comment:
yeah this is a very good point. I thought about this while making changes as well. I guess we need to pull in @ericmjonas for this :)

@ericmjonas do you have an idea how many changes we made when we forked from cloudpickle ? Also do you see us syncing their more recent changes ?